gems/sorbet/lib/find-gem-rbis.rb

```diff
@@ -37,7 +37,7 @@ def self.main
     gem_source_paths = T.let([], T::Array[String])
     gemspecs.each do |gemspec|
       gem_source_paths << paths_within_gem_sources(gemspec)
-    end
+    end unless gemspecs.any? { |g| g.name == 'tapioca' }
```

Comment:
I'm fine with this change. Can you make this a little more explicit by refactoring it to use a prefix `unless` (`unless; ...; end` instead of a postfix `unless`, and add a bit of a summary from your well written PR description explaining why this is here?

Reply:
Good idea. Done.